### PR TITLE
Added EEPROM

### PR DESCRIPTION
**Pull request by RaspberryPiFan**

### CODE REVIEW

File: raspinfo/raspinfo [hunk 1]
```diff
@@ -109,3 +109,11 @@ echo "---------"
 echo
 
 dmesg | sed -e "s/\([0-9a-fA-F]\{1,4\}:\)\{7,7\}[0-9a-fA-F]\{1,4\}/y.y.y.y.y.y.y.y/g" | sed -e "s/[0-9a-fA-F]\{1,4\}:\(:[0-9a-fA-F]\{1,4\}\)\{1,4\}/y::y.y.y.y/g" | sed -e "s/\([0-9a-fA-F]\{2,2\}\:\)\{5,5\}[0-9a-fA-F]\{2,2\}/m.m.m.m/g"
+
+
+echo
+echo "eeprom (RPi 4)"
```
Comment (pelwell) [inline raspinfo/raspinfo]: Can we capitalise "EEPROM"?

Comment (pelwell) [reply raspinfo/raspinfo]: And add a few more dashes to underline it?

Comment (RaspberryPiFan) [reply raspinfo/raspinfo]: Done

**Comment by pelwell** [comment]
That looks good as far as it goes. My only remaining concern is about the behaviour on other devices - I think it would be better to skip that section altogether for Pis other than the 4B.

**Comment by andrum99** [comment]
On all but the Pi 4B, rpi-eeprom-update runs and does nothing, so it's safe. (The first thing it does is check it's running on a Pi 4B and exits if it isn't).

**Comment by pelwell** [comment]
An empty Pi 4 section on a Pi 3 seems a bit ugly. What do you think, @JamesH65 ?

**Comment by JamesH65** [comment]
@RaspberryPiFan Any reason why the script cannot check if its on a Pi4 and only add that section if it is?

**Comment by RaspberryPiFan** [comment]
@JamesH65 @pelwell I added a check.

**Comment by pelwell** [comment]
Thanks - I'm happy with that.

**Pull request merged**